ts/client/scripts/create-gov-ix.ts

```diff
@@ -381,15 +382,45 @@ async function createMangoAccount(): Promise<void> {
   console.log(await serializeInstructionToBase64(ix));
 }
 
+async function idlResize(): Promise<void> {
+  const idlIxBytes = [0x40, 0xf4, 0xbc, 0x78, 0xa7, 0xe9, 0x69, 0x0a];
```

Comment:
nit: can you link to some code where you grabbed these? or was this from some real txs?

Reply:
that's just the anchor constant for idl instruction calls, you can find it in the anchor source or in instructions that set idl etc